### DISCUSSION TITLE
Sort order of bookmarks

### DISCUSSION THREAD

**Discussion by tim-field** [alefragnani/vscode-bookmarks#643] [Ideas]
I often add bookmarks as a breadcrumb while I follow through code execution.  But the bookmarks aren't sorted in the order that they were created, so the trail is confused.

**Comment by alefragnani** [comment]
Hi @tim-field ,

Yes, the bookmarks does not store a timestamp when are toggled so, there is no way (today) to know the _order_ which the bookmarks were toggled.

There is an open issue requesting sorting for recent (#183). I suggest you to upvote and add aditional comments, if necessary.

Hope this helps
